jdt/Formatter.java

```diff
@@ -0,0 +1,58 @@
+import java.io.File;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Formatter {
+
+	private static final List<String> PREFIXES = List.of(
```

Comment:
These are needed, even if the full (repo-relative) paths are given in the command?

```
run_restyler jdt ./src/main/java/ShouldFormat1.java ./src/main/java/ShouldFormat2.java
```

Is it possible/common to use other prefixes in Java projects, and would that just get ignored by this Restyler?